content/docs/user-guide/assets/pipeline/asset-cache.md

```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
 
 The Asset Database is only used by Asset Processor. The Asset Database is a development time database containing details about assets, jobs, **Asset Builders**, and scan directories. It contains all the information Asset Processor requires to determine which assets have been processed, and which assets need to be processed.
 
-With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, remembering all the pending process jobs in the job queue.
+With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, rebuilding the queue of jobs to process.

```

Comment:
"rebuilding" here sounds like re-processing existing assets. Is that your intent?

Reply:
I've re-worded this.